src/Daemon.ts

```diff
@@ -1,40 +1,35 @@
-import { createServer } from 'http2'
-import { Server } from 'net'
+/* Imports for SSH 'server implementation */
+import * as ssh2 from 'ssh2'
+import * as net from 'net'
+import * as fs from 'fs'
+/* Variables for hostkeys */
+// var crypto = require('crypto');
+// var inspect = require('util').inspect;
 
-const MAX_DELAY_MS = 2000
+// var ssh2 = require('ssh2');
+// var utils = ssh2.utils;
 
-/**
- * Randomly generate a number between 0 and {@link MAX_DELAY_MS}.
- * @returns the generated number
- */
-function mockDelayMs(): number {
-	return MAX_DELAY_MS * Math.random()
-}
-
-/**
- * Dummy Junknet server, using HTTP/2 as a placeholder.
- * Echoes the requested path back to the client in uppercase as plain text.
- * Randomly delayed response to create more interesting tests.
- * @experimental
- */
-export function createDaemon(): Server {
-	const server = createServer()
-	server.on('stream', (stream, headers) => {
-		const input = headers[':path']
-		if (!input) {
-			// Can never happen, but let's make the type checker happy.
-			return
-		}
+export function createDaemon(): net.Server {
+	// Create server object
+	const server = new ssh2.Server(
+		// SSH authentication information
+		{ hostKeys: [fs.readFileSync('host.key')] },
```

Comment:
I'll look into that! Rohit and I were discussing this when I wrote the code

Reply:
This would be a nice feature to have, but relatively low priority. We can always add this later, probably when it comes time to write integration tests for the daemon.